src/Oscoin/CLI/KeyStore.hs

```diff
@@ -61,6 +63,21 @@ ensureConfigDir mbUserPath = do
     configDir <- getConfigPath mbUserPath
     liftIO $ createDirectoryIfMissing True configDir
 
+data KeyStoreError =
+    PrivateKeyDeserialisationFailure DeserialiseFailure
+
+instance Show KeyStoreError where
+    show = \case
+      PrivateKeyDeserialisationFailure cborError ->
+          mconcat [
+                    "Loading the oscoin's private key from the configuration "
```

Comment:
```suggestion
                    "Loading the private key from the configuration "
```

Reply:
Actually, it might be useful to stick the path to the "configuration folder" in the `PrivateKeyDeserialisationFailure` ctor

Reply:
True, let me do that!